core/packages/contracts/.prettierrc

```diff
@@ -7,7 +7,8 @@
           "tabWidth": 4,
           "singleQuote": false,
           "bracketSpacing": true,
-          "explicitTypes": "always"
+          "explicitTypes": "always",
+          "plugins": ["prettier-plugin-solidity"]
```

Comment:
nice!

Reply:
@vgeddes  conflict resolved, no logic but only format change in this pr and also add a git hook for it when commit